### ISSUE TITLE
Collect IIIF Presentations on website

### ISSUE THREAD

**Issue by glenrobson** [IIIF/iiif-root-website#13]
From https://github.com/IIIF/iiif.io/issues/887:

How about the following structure:
- /about/presentations/
  - introduction
    - overview
    - usecases
  - api
    - image
    - presentation
    - search
    - auth
  - software
    - client
    - server
  - content
    - manuscripts
    - newspapers
    - museums
  - model
    - linked-data
    - shared-canvas
    - open-annotation
  - related

Do community groups get a space here, or do they have more fine grained control of their own community/groups/ space, and can put presentations there as desired?  How would one find presentations on newspapers in IIIF, for example?  I've put them here for now but that could be just a symlink or it could be a more finely curated set.

**Comment by glenrobson** [comment]
Rob: "Also the various Youtube videos from outreach events and other recorded conferences / webinars etc."

**Comment by mprove** [comment]
yes. It would support trainers and teachers of IIIF. It would be easier for them to create their own training courses if they can easily peek into existing materials. 
/quoting myself from the Outreach channel on Slack  https://iiif.slack.com/archives/C6AE311FD/p1696526104130579
